reliability-v2/tasks/TaskManager.py

```diff
@@ -134,11 +133,16 @@ def run_users_tasks(self,group):
         users_tasks = []
         # todo: validate
         if persona == "admin":
+            users = group.get("uses",1)
             users_tasks.append({"user":"kubeadmin","group_name":name,"loops":loops,"trigger":trigger,"interval":interval,"jitter":jitter,"tasks":tasks})
+            self.logger.info(f"Will run group '{name}' with admin user in {users} sessions concurrently")
         elif persona == "developer":
-            for i in range(users):
+            user_start = group.get("user_start",0)
+            user_end = group.get("user_end",15)
+            for i in range(user_start, user_end):
                 users_tasks.append({"user":f"testuser-{i}","group_name":name,"loops":loops,"trigger":trigger,"interval":interval,"jitter":jitter,"tasks":tasks})
-        self.logger.info(f"Will run group '{name}' with {users} users concurrently")
+            users = user_end - user_start
+            self.logger.info(f"Will run group '{name}' with {users} users concurrently")
```

Comment:
It's different for different groups. e.g. for developer-1 it will run with 15 users(0-14), while for developer-2, it will run with 1 user(15), because those groups are run with concurrent thread pools, they are independence to each other.
https://github.com/openshift/svt/blob/38bbf45053c8469822009915e6c2778b459355ae/reliability-v2/tasks/TaskManager.py#L155-L158

Reply:
For the log content, its more for debugging purpose. 
There are more readable logs for each group after it, like
```
2023-02-28 08:55:37,248 - INFO - Will run group 'admin-1' with admin user in 1 sessions concurrently
2023-02-28 08:55:37,249 - INFO - Will run group 'developer-1' with 15 users concurrently
2023-02-28 08:55:37,249 - INFO - Will run group 'developer-2' with 1 users concurrently
```